### DISCUSSION TITLE
View On Github Button on CS

### DISCUSSION THREAD

**Discussion by ka1nsha** [community/community#19573] [Other Feature Feedback, Questions, & Ideas] [answered]
When we click to code snippet for viewing,  We should be able to view the repository on Github(not cs.github) with clicking to an additional button.

**Comment by rilysh** [comment]
@ka1nsha here cs means codespace, codespace works completely different way. Codespace base URL: https://cs.github.com will open codespace for more information https://docs.github.com/en/codespaces

Hope this clarifies.

**Reply by ka1nsha** [reply]
Actually cs means codesearch(Beta program)

**Comment by look** [comment, marked as answer]
Thanks for the feedback @Ruzie.

While there is not a way right now to go directly to GitHub.com from the cs.github.com results list, if you click on a result there is a "view on github.com" link at the top.

**Comment by ka1nsha** [comment]
Oh okay. I saw this button (View Repo) at the top on cs.github.com thank you.
